doc/code/qml_fourier.rst

```diff
@@ -0,0 +1,129 @@
+qml.fourier
+===========
+
+This module contains methods for computing Fourier series representations of
+quantum circuits.
+
+Fourier series of quantum circuits
+----------------------------------
+
+Consider a quantum circuit that depends on a parameter vector :math:`x` with
+length :math:`N`. The circuit involves application of some unitary operations
+:math:`U(x)`, and then measurement of a particular expectation value
```

Comment:
@glassnotes: we need to add that `U(x)` has to encode each feature of x into single-param gates